restapi/src/main/java/gov/cdc/sdp/cbr/restapi/InputController.java

```diff
@@ -21,52 +22,69 @@
 import com.google.gson.Gson;
 import com.google.gson.JsonSyntaxException;
 
+import gov.cdc.sdp.cbr.trace.TraceService;
+import gov.cdc.sdp.cbr.trace.model.TraceStatus;
+
 @Controller
 public class InputController {
 
     @Autowired
     protected CamelContext camelContext;
     
+    @Autowired
+    private TraceService traceService;
+    
     @Value("${input.post.endpoint}")
     public String endpoint;
     
     @RequestMapping(value="/cbr/input", method = RequestMethod.POST)
     
-    public @ResponseBody String doSomething(
+    public @ResponseBody String processInputFile (
                 @RequestParam("id") String id,
                 @RequestParam("source") String source,
+                // TODO: Add createdAt, change createdAt in transformer to receivedAt?
                 @RequestParam("metadata") String jsonMetadata,
                 @RequestParam("file") MultipartFile file)  {
+        
         String cbrId = "CBR_" + source + "_" + id;
         
-        // TODO: Log message in
+        try {
+			traceService.addTraceMessage(cbrId, source, TraceStatus.INFO, "Message from id " + id + " received");
+		} catch (SQLException e) {
+			e.printStackTrace();
```

Comment:
Please do not just catch and print an exception. Either re-throw or find a better way to log this issue.